### ISSUE TITLE
Widen EmptyState description PropType to accept ReactNode

### ISSUE THREAD

**Issue by brandongregoryscott** [segmentio/evergreen#1447]
Right now the `description` PropType is limited to a string and will output console warnings in development if JSX is provided (such as a `<Text>` or `<Strong>` component for styling), but it renders fine:

![image](https://user-images.githubusercontent.com/11774799/163799931-d9f3417e-a75e-444a-b27b-b1dd2b31bfdf.png)

```
Warning: Failed prop type: Invalid prop `description` of type `object` supplied to `EmptyState`, expected `string`.
    at EmptyState (http://localhost:3000/static/js/vendors~main.chunk.js:25400:25)
```

Other components (such as the `FormField` `label`, `description` and `hint` accept `PropTypes.node`), so I don't see the harm in widening it here too!

**Comment by brandongregoryscott** [comment]
Fixed via #1460 and released in [v6.9.7](https://github.com/segmentio/evergreen/releases/tag/v6.9.7)

**Comment by brandongregoryscott** [comment]
Looks like we forgot to update the TypeScript definitions for `EmptyStateOwnProps` which still only accepts `description` as a string! oops.
